File: presentation/src/test/java/android/util/Base64.java
/*
 * carrot-android © 2022 Ji Sungbin. all rights reserved.
 * carrot-android license is under the MIT.
 *
 * [Base64.java] created by Ji Sungbin on 22. 1. 28. 오후 10:28
 *
 * Please see: https://github.com/carrot-style/carrot-android/blob/main/LICENSE.
 */

package android.util;

public class Base64 {
    public static String encodeToString(byte[] input, int flags) {
        return java.util.Base64.getEncoder().encodeToString(input);
    }

    public static byte[] decode(String str, int flags) {
        return java.util.Base64.getDecoder().decode(str);
    }
}

File: data/src/main/kotlin/style/carrot/android/data/util/base64.kt
/*
 * carrot-android © 2022 Ji Sungbin. all rights reserved.
 * carrot-android license is under the MIT.
 *
 * [base64.kt] created by Ji Sungbin on 22. 1. 23. 오후 12:40
 *
 * Please see: https://github.com/carrot-style/carrot-android/blob/main/LICENSE.
 */

package style.carrot.android.data.util

import android.util.Base64

fun String.toBase64() = Base64.encodeToString(toByteArray(), Base64.NO_WRAP)!!
